aten/src/THC/generic/THCTensorScatterGather.cu

```diff
@@ -9,6 +9,11 @@
 
 void THCTensor_(gather)(THCState* state, THCTensor *tensor,
                          THCTensor *src, int dim, THCudaLongTensor *index) {
+  // TODO (@zasdfgbnm): remove this.
```

Comment:
is it used in the inner loop?  If not, you can call the aten function if you wrapper the THCTensors via THTensor_wrap.

Reply:
Thanks for the information. Now I removed the legacy implementation using the `THTensor_wrap`